core/log.go

```diff
@@ -66,7 +67,18 @@ func (l *Logger) Log(level int, format string, args ...interface{}) {
 		payload := fmt.Sprintf(session.Config.WebhookPayload, text)
 		http.Post(session.Config.Webhook, "application/json", strings.NewReader(payload))
 	}
+	
+	if session.Config.Logstash != "" {
+		text := colorStrip(fmt.Sprintf(format, args...))
+		payload := fmt.Sprintf(session.Config.Logstash, text)
+  		pc, err := net.ListenPacket("udp4", ":" + session.Config.LogstashPort)
+  		defer pc.Close()
 
+  		addr, err := net.ResolveUDPAddr("udp4", session.Config.Logstash)
```

Comment:
Did you test that? It seems like `err` is defined but never used and might give an error when compiling

Reply:
Sadly no, because I had a couple of minutes so I just made the change and fatally assumed it would be fine.  Is there a preferred way you want the err message to be handled?

Reply:
```suggestion
  		addr, err := net.ResolveUDPAddr("udp4", session.Config.Logstash)
  		if err != nil {
  		  log.Error("Some meaningful message")
  		 }
```